tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
+    # Example 1: Two tensors can be combined into one Dataset object.
```

Comment:
maybe remove the "Example 1: " part. The lines above are also examples, so starting the numbering at 1 here looks strange.